data/campaigns/Eastern_Invasion/scenarios/11_Captured.cfg

```diff
@@ -743,6 +747,7 @@
             {STORE_UNIT_VAR (role=escapee sidekick 2) name deserter_name}
             [message]
                 role=escapee sidekick 1
+                #po: The deserter, "escapee sidekick 2", will always be male. The other escapees are random units from the player's recall list
```

Comment:
Seems like this also needs a `female_message`?

Reply:
Thought this was the same as the other one, and then realised that it's a new string. Will fix (but it's not in the first force-pushed update).